packages/starlight/components/Pagination.astro

```diff
@@ -38,8 +38,9 @@ const t = useTranslations(locale);
 
 <style>
 	.pagination-links {
-		flex-wrap: wrap;
-		gap: 1rem;
+		display: grid;
+		grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
```

Comment:
Played around a bit and I think a slightly higher value here could make sense (`18rem` is `288px`). I wondered if this should be linked to other variables (e.g. `calc(var(--sl-content-width) / 3)` or something), but I think the target size is independent of anything else — if someone sets a larger content width, they probably still want these links to shrink at smaller viewports and not break earlier. So I think setting a value here is fine. (Could also be pulled out into a variable in `props.css`, e.g. `--sl-min-pagination-link-width` or something.)

We should also wrap the target value in `min()` so that it never exceeds 100% of the container. (One quirk of grid layout is that `1fr` can be MORE than `100%` because in the case where there’s just one grid item, the “free” space is calculated based on our minimum size.)

```suggestion
		grid-template-columns: repeat(auto-fit, minmax(min(18rem, 100%), 1fr));
```

Reply:
> I wondered if this should be linked to other variables (e.g. calc(var(--sl-content-width) / 3) or something), but I think the target size is independent of anything else

Yep, I tried to find some logic based on other custom properties as well but couldn't find anything related.

> We should also wrap the target value in min() so that it never exceeds 100% of the container. (One quirk of grid layout is that 1fr can be MORE than 100% because in the case where there’s just one grid item, the “free” space is calculated based on our minimum size.)

Oh yeah, good catch! Let's do that!